frontend/components/receipts/utils/order/order-size.spec.tsx

```diff
@@ -0,0 +1,21 @@
+import { screen, render } from '@testing-library/react'
+import { OrderSizeComponent } from './order-size'
+import { locators as sizeWithTooltipLocators } from '../string-amounts/size-with-tooltip'
+import { locators as amountWithSymbolLocators } from '../string-amounts/amount-with-symbol'
+
+describe('OrderSizeComponent', () => {
+  it('should return basic data if markets are loading or formattedSize or symbol is not defined', () => {
+    render(
+      <OrderSizeComponent marketsLoading={true} size="100" marketId="someMarketId" formattedSize="100" symbol="BTC" />
+    )
+    expect(screen.getByTestId(sizeWithTooltipLocators.sizeWithTooltip)).toBeInTheDocument()
+  })
+
+  // 1118-ORDS-012 I see enriched data for the order size when data has been loaded appropriately
```

Comment:
tiny change but can we put the ac in the first line of the test? This is what we do everywhere else

Reply:
Dex asked me to put them at the top of the appropriate test. Please discuss between yourselves which is the right approach